### PR TITLE
Add issue markers for outdated diagrams

### PR DESCRIPTION
**Pull request by OR13**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1122.html" title="Last updated on May 9, 2023, 11:22 PM UTC (962405f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1122/80cefee...962405f.html" title="Last updated on May 9, 2023, 11:22 PM UTC (962405f)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Review by msporny — APPROVED** [review]
Approving, noting that the diagrams are fixed in PR #1129, so we might just want to do that PR instead?

**Comment by OR13** [comment]
PR has been open for 1 week, I prefer to merge it.  

@brentzundel @Sakurann can you comment on @msporny 's proposal, to take the other PR instead?

**Comment by msporny** [comment]
> @OR13 if PR #1129 is merged would you still want issue markers on the diagrams?

Waiting on a response to this before merging, @OR13.

**Pull request closed without merging**
